src/commands/commit/prompts.js

```diff
@@ -25,11 +24,15 @@ export type Answers = {
   message: string
 }
 
-export default (gitmojis: Array<Gitmoji>, mode: CommitMode): Array<Object> => {
-  const { title, message } = getDefaultCommitContent(mode)
+export default (
+  gitmojis: Array<Gitmoji>,
+  { title, message }: CommitContent,
+  defaultAnswers: $Shape<Answers>
+): Array<Object> => {
+  const questions = []
 
-  return [
-    {
+  if (!defaultAnswers.gitmoji) {
```

Comment:
Can we stick to the old format? I just want to keep immutability in the codebase as much as possible 🙏🏼

Reply:
Sure